python/ray/data/dataset.py

```diff
@@ -1941,7 +1943,9 @@ def write_numpy(
             path: The path to the destination root directory, where npy
                 files will be written to.
             column: The name of the table column that contains the tensor to
-                be written. This defaults to "value".
+                be written. The default is the column name that Datasets uses for
+                storing tensors in single-column tables. See
+                ``ray.data.impl.arrow_block.VALUE_COL_NAME`` for the exact name.
```

Comment:
Could we just say __value__ instead of indirecting here? It's not going to change so this is just making it slightly harder for the user to read.

Reply:
Yeah good point, this should be pretty stable.